docs/data-sourcing-and-transformation.md

```diff
@@ -0,0 +1,91 @@
+# Data Sourcing and Transformations
+
+![alpha](assets/alpha.svg)
+
+> v3.0 and after
```

Comment:
v3.1

Reply:
Oh right, I got ahead of myself... should we consider launching with 3.0? Even if it's bare bones

Reply:
I think we should hold on this (same for conditional artifacts and container set template) to v3.1 so we can focus on bug fixes for v3.0

Reply:
Sure